HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,47 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is a platform specific path separator.
+
+method::isVowel
+
+returns:: true if the char is a vowel.
+
+method::isControl
+
+returns:: true if the char is a control character.
+
+method::asAscii
+
+returns:: the object itself.
+
+method::asUnicode
+
+returns:: a single Unicode value. Ascii is a subset of Unicode.
+
+method::hash
+
+returns:: a code used to index into a hash table.
+
+method::++
+
+A concatenation of the two chars. The result is link::Classes/String::
+
+method::<
+
+True if the receiver is alphabetically sooner. It is comparing the Unicode values. Uppercase characters have lower Unicode value than lowercase characters.
```

Comment:
how about something like: `returns True if the Unicode value of the receiver is less that the Unicode value of the argument. While this is equivalent to alphabetical order when comparing for example two lowercase chars, note that uppercase characters have lower Unicode values that lowercase characters, and digits have lower than either.`